indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -908,7 +908,9 @@ public void tryInit()
                     notice.handle();
                     Instant handleNoticeEndTime = Instant.now();
                     Duration timeElapsed = Duration.between(handleNoticeStartTime, handleNoticeEndTime);
-                    log.debug("Handled notice [%s] from notices queue in [%d] ms, current notices queue size [%d]", notice.getClass().getName(), timeElapsed.toMillis(), getNoticesQueueSize());
+                    String noticeType = notice.getClass().getName();
```

Comment:
instead of relying on the class name, which could change, I would suggest we use constants like we do in `Task.getType()`. This also ensures the dimension values are consistently formatted (I believe the convention is `lowercase` or `snake_lowercase` for most of our metric dimensions) rather than adding a new convention.